cs_misp_import/intel_client.py

```diff
@@ -59,7 +59,9 @@ def get_reports(self, start_time):
             offset += resp_json.get('meta', {}).get('pagination', {}).get('limit', 5000)
             first_run = False
 
-            reports.extend(resp_json.get('resources', []))
+            resources = resp_json.get('resources', [])
+            if resources is not None and len(resources) > 0:
```

Comment:
We should be able to get away with `if resources:` here. That would return a False if resources is a None or empty.